app/Http/Controllers/SampleController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Mail;
+use App\Mail\SampleNotification;
+
+class SampleController extends Controller
+{
+  public function SampleNotification()
+  {
+    $name = auth()->user()->name;
+    $text = auth()->user()->name;
+    $to = auth()->user()->D_email;
+
+    Mail::to($to)->send(new SampleNotification($name, $text));
```

Comment:
Please change the name of your SampleNotification, use something that describe what it does, like EmailNotification or something else.